diff --git a/panels/network/wireless-security/wireless-security.c b/panels/network/wireless-security/wireless-security.c
index 23b8a27..bd233b0 100644
--- a/panels/network/wireless-security/wireless-security.c
+++ b/panels/network/wireless-security/wireless-security.c
@@ -25,10 +25,25 @@
 #include <string.h>
 
 #include "wireless-security.h"
+#include "wireless-security-resources.h"
 #include "eap-method.h"
 #include "utils.h"
 
-G_DEFINE_BOXED_TYPE (WirelessSecurity, wireless_security, wireless_security_ref, wireless_security_unref)
+GType
+wireless_security_get_type (void)
+{
+	static GType type_id = 0;
+
+	if (!type_id) {
+		g_resources_register (wireless_security_get_resource ());
+
+		type_id = g_boxed_type_register_static ("WirelessSecurity",
+							(GBoxedCopyFunc) wireless_security_ref,
+							(GBoxedFreeFunc) wireless_security_unref);
+	}
+
+	return type_id;
+}
 
 GtkWidget *
 wireless_security_get_widget (WirelessSecurity *sec)
diff --git a/panels/network/wireless-security/wireless-security.h b/panels/network/wireless-security/wireless-security.h
index cb6553be5..4de618e16 100644
--- a/panels/network/wireless-security/wireless-security.h
+++ b/panels/network/wireless-security/wireless-security.h
@@ -23,6 +23,8 @@
 #ifndef WIRELESS_SECURITY_H
 #define WIRELESS_SECURITY_H
 
+#include <gtk/gtk.h>
+
 typedef struct _WirelessSecurity WirelessSecurity;
 
 typedef void (*WSChangedFunc) (WirelessSecurity *sec, gpointer user_data);
