diff --git a/panels/network/wireless-security/eap-method-fast.ui b/panels/network/wireless-security/eap-method-fast.ui
index 4b19a66..5307ddb 100644
--- a/panels/network/wireless-security/eap-method-fast.ui
+++ b/panels/network/wireless-security/eap-method-fast.ui
@@ -46,8 +46,8 @@
           <object class="GtkLabel" id="eap_fast_anon_identity_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Anony_mous identity:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Anony_mous identity</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_fast_anon_identity_entry</property>
           </object>
@@ -72,8 +72,8 @@
           <object class="GtkLabel" id="eap_fast_pac_file_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">PAC _file:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">PAC _file</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_fast_pac_file_button</property>
           </object>
@@ -117,8 +117,8 @@
           <object class="GtkLabel" id="eap_fast_inner_auth_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Inner authentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Inner authentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_fast_inner_auth_combo</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-leap.ui b/panels/network/wireless-security/eap-method-leap.ui
index 1d1f23d..d4c0625 100644
--- a/panels/network/wireless-security/eap-method-leap.ui
+++ b/panels/network/wireless-security/eap-method-leap.ui
@@ -22,8 +22,8 @@
           <object class="GtkLabel" id="eap_leap_username_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Username:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Username</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_leap_username_entry</property>
           </object>
@@ -36,8 +36,8 @@
           <object class="GtkLabel" id="eap_leap_password_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Password:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Password</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_leap_password_entry</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-peap.ui b/panels/network/wireless-security/eap-method-peap.ui
index a97123a..2f8b8cd 100644
--- a/panels/network/wireless-security/eap-method-peap.ui
+++ b/panels/network/wireless-security/eap-method-peap.ui
@@ -46,8 +46,8 @@
           <object class="GtkLabel" id="eap_peap_anon_identity_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Anony_mous identity:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Anony_mous identity</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_peap_anon_identity_entry</property>
           </object>
@@ -72,8 +72,8 @@
           <object class="GtkLabel" id="eap_peap_ca_cert_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">C_A certificate:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">C_A certificate</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_peap_ca_cert_button</property>
           </object>
@@ -135,8 +135,8 @@
           <object class="GtkLabel" id="eap_peap_inner_auth_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Inner authentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Inner authentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_peap_inner_auth_combo</property>
           </object>
@@ -172,8 +172,8 @@
           <object class="GtkLabel" id="eap_peap_version_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">PEAP _version:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">PEAP _version</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_peap_version_combo</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-simple.ui b/panels/network/wireless-security/eap-method-simple.ui
index b3318e4..7d95d28 100644
--- a/panels/network/wireless-security/eap-method-simple.ui
+++ b/panels/network/wireless-security/eap-method-simple.ui
@@ -22,8 +22,8 @@
           <object class="GtkLabel" id="eap_simple_username_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Username:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Username</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_simple_username_entry</property>
           </object>
@@ -36,8 +36,8 @@
           <object class="GtkLabel" id="eap_simple_password_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Password:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Password</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_simple_password_entry</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-tls.ui b/panels/network/wireless-security/eap-method-tls.ui
index 5557593..3c6291b 100644
--- a/panels/network/wireless-security/eap-method-tls.ui
+++ b/panels/network/wireless-security/eap-method-tls.ui
@@ -22,8 +22,8 @@
           <object class="GtkLabel" id="eap_tls_identity_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">I_dentity:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">I_dentity</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_tls_identity_entry</property>
           </object>
@@ -48,8 +48,8 @@
           <object class="GtkLabel" id="eap_tls_user_cert_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_User certificate:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_User certificate</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_tls_user_cert_button</property>
           </object>
@@ -64,8 +64,8 @@
           <object class="GtkLabel" id="eap_tls_ca_cert_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">C_A certificate:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">C_A certificate</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_tls_ca_cert_button</property>
           </object>
@@ -112,8 +112,8 @@
           <object class="GtkLabel" id="eap_tls_private_key_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Private _key:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Private _key</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_tls_private_key_button</property>
           </object>
@@ -142,8 +142,8 @@
           <object class="GtkLabel" id="eap_tls_private_key_password_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Private key password:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Private key password</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_tls_private_key_password_entry</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-ttls.ui b/panels/network/wireless-security/eap-method-ttls.ui
index ce00c26..70db9ed 100644
--- a/panels/network/wireless-security/eap-method-ttls.ui
+++ b/panels/network/wireless-security/eap-method-ttls.ui
@@ -29,8 +29,8 @@
           <object class="GtkLabel" id="eap_ttls_anon_identity_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Anony_mous identity:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Anony_mous identity</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_ttls_anon_identity_entry</property>
           </object>
@@ -55,8 +55,8 @@
           <object class="GtkLabel" id="eap_ttls_ca_cert_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">C_A certificate:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">C_A certificate</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_ttls_ca_cert_button</property>
           </object>
@@ -103,8 +103,8 @@
           <object class="GtkLabel" id="eap_ttls_inner_auth_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Inner authentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Inner authentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">eap_ttls_inner_auth_combo</property>
           </object>
diff --git a/panels/network/wireless-security/ws-dynamic-wep.ui b/panels/network/wireless-security/ws-dynamic-wep.ui
index 4bd8520..cf31f12 100644
--- a/panels/network/wireless-security/ws-dynamic-wep.ui
+++ b/panels/network/wireless-security/ws-dynamic-wep.ui
@@ -35,8 +35,8 @@
           <object class="GtkLabel" id="dynamic_wep_auth_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Au_thentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Au_thentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">dynamic_wep_auth_combo</property>
           </object>
diff --git a/panels/network/wireless-security/ws-leap.ui b/panels/network/wireless-security/ws-leap.ui
index dc936ad..b4d9bec 100644
--- a/panels/network/wireless-security/ws-leap.ui
+++ b/panels/network/wireless-security/ws-leap.ui
@@ -22,8 +22,8 @@
           <object class="GtkLabel" id="leap_username_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Username:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Username</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">leap_username_entry</property>
           </object>
@@ -36,8 +36,8 @@
           <object class="GtkLabel" id="leap_password_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Password:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Password</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">leap_password_entry</property>
           </object>
diff --git a/panels/network/wireless-security/ws-wep-key.ui b/panels/network/wireless-security/ws-wep-key.ui
index 62b11a5..c0e5f55 100644
--- a/panels/network/wireless-security/ws-wep-key.ui
+++ b/panels/network/wireless-security/ws-wep-key.ui
@@ -52,8 +52,8 @@
           <object class="GtkLabel" id="wep_key_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Key:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Key</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">wep_key_entry</property>
           </object>
@@ -111,8 +111,8 @@
           <object class="GtkLabel" id="auth_method_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Au_thentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Au_thentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">auth_method_combo</property>
           </object>
@@ -148,8 +148,8 @@
           <object class="GtkLabel" id="key_index_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">WEP inde_x:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">WEP inde_x</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">key_index_combo</property>
           </object>
diff --git a/panels/network/wireless-security/ws-wpa-eap.ui b/panels/network/wireless-security/ws-wpa-eap.ui
index 2da2148..942af88 100644
--- a/panels/network/wireless-security/ws-wpa-eap.ui
+++ b/panels/network/wireless-security/ws-wpa-eap.ui
@@ -29,8 +29,8 @@
           <object class="GtkLabel" id="wpa_eap_auth_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">Au_thentication:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">Au_thentication</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">wpa_eap_auth_combo</property>
           </object>
diff --git a/panels/network/wireless-security/ws-wpa-psk.ui b/panels/network/wireless-security/ws-wpa-psk.ui
index 4ec6909..b34720a 100644
--- a/panels/network/wireless-security/ws-wpa-psk.ui
+++ b/panels/network/wireless-security/ws-wpa-psk.ui
@@ -18,8 +18,8 @@
           <object class="GtkLabel" id="wpa_psk_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Password:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Password</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">wpa_psk_entry</property>
           </object>
@@ -46,8 +46,8 @@
           <object class="GtkLabel" id="wpa_psk_type_label">
             <property name="visible">True</property>
             <property name="can_focus">False</property>
-            <property name="xalign">0</property>
-            <property name="label" translatable="yes">_Type:</property>
+            <property name="xalign">1</property>
+            <property name="label" translatable="yes">_Type</property>
             <property name="use_underline">True</property>
             <property name="mnemonic_widget">wpa_psk_type_combo</property>
           </object>
diff --git a/panels/network/wireless-security/eap-method-fast.ui b/panels/network/wireless-security/eap-method-fast.ui
index 5307ddb..36e74c3 100644
--- a/panels/network/wireless-security/eap-method-fast.ui
+++ b/panels/network/wireless-security/eap-method-fast.ui
@@ -40,7 +40,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">5</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="eap_fast_anon_identity_label">
diff --git a/panels/network/wireless-security/eap-method-leap.ui b/panels/network/wireless-security/eap-method-leap.ui
index d4c0625..c4de1dc 100644
--- a/panels/network/wireless-security/eap-method-leap.ui
+++ b/panels/network/wireless-security/eap-method-leap.ui
@@ -13,7 +13,7 @@
         <property name="valign">start</property>
         <property name="n_rows">3</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <placeholder/>
diff --git a/panels/network/wireless-security/eap-method-peap.ui b/panels/network/wireless-security/eap-method-peap.ui
index 2f8b8cd..d16e00a 100644
--- a/panels/network/wireless-security/eap-method-peap.ui
+++ b/panels/network/wireless-security/eap-method-peap.ui
@@ -40,7 +40,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">5</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="eap_peap_anon_identity_label">
diff --git a/panels/network/wireless-security/eap-method-simple.ui b/panels/network/wireless-security/eap-method-simple.ui
index 7d95d28..af00cb9 100644
--- a/panels/network/wireless-security/eap-method-simple.ui
+++ b/panels/network/wireless-security/eap-method-simple.ui
@@ -13,7 +13,7 @@
         <property name="valign">start</property>
         <property name="n_rows">3</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <placeholder/>
diff --git a/panels/network/wireless-security/eap-method-tls.ui b/panels/network/wireless-security/eap-method-tls.ui
index 3c6291b..0079715 100644
--- a/panels/network/wireless-security/eap-method-tls.ui
+++ b/panels/network/wireless-security/eap-method-tls.ui
@@ -13,7 +13,7 @@
         <property name="valign">start</property>
         <property name="n_rows">6</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <placeholder/>
diff --git a/panels/network/wireless-security/eap-method-ttls.ui b/panels/network/wireless-security/eap-method-ttls.ui
index 70db9ed..df942d9 100644
--- a/panels/network/wireless-security/eap-method-ttls.ui
+++ b/panels/network/wireless-security/eap-method-ttls.ui
@@ -23,7 +23,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">4</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="eap_ttls_anon_identity_label">
diff --git a/panels/network/wireless-security/ws-dynamic-wep.ui b/panels/network/wireless-security/ws-dynamic-wep.ui
index cf31f12..d6bc12b 100644
--- a/panels/network/wireless-security/ws-dynamic-wep.ui
+++ b/panels/network/wireless-security/ws-dynamic-wep.ui
@@ -23,7 +23,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">3</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <placeholder/>
diff --git a/panels/network/wireless-security/ws-leap.ui b/panels/network/wireless-security/ws-leap.ui
index b4d9bec..06cb645 100644
--- a/panels/network/wireless-security/ws-leap.ui
+++ b/panels/network/wireless-security/ws-leap.ui
@@ -13,7 +13,7 @@
         <property name="valign">start</property>
         <property name="n_rows">3</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <placeholder/>
diff --git a/panels/network/wireless-security/ws-wep-key.ui b/panels/network/wireless-security/ws-wep-key.ui
index c0e5f55..a4e14dc 100644
--- a/panels/network/wireless-security/ws-wep-key.ui
+++ b/panels/network/wireless-security/ws-wep-key.ui
@@ -46,7 +46,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">4</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="wep_key_label">
diff --git a/panels/network/wireless-security/ws-wpa-eap.ui b/panels/network/wireless-security/ws-wpa-eap.ui
index 942af88..5d71936 100644
--- a/panels/network/wireless-security/ws-wpa-eap.ui
+++ b/panels/network/wireless-security/ws-wpa-eap.ui
@@ -23,7 +23,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">2</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="wpa_eap_auth_label">
diff --git a/panels/network/wireless-security/ws-wpa-psk.ui b/panels/network/wireless-security/ws-wpa-psk.ui
index b34720a..3c689d3 100644
--- a/panels/network/wireless-security/ws-wpa-psk.ui
+++ b/panels/network/wireless-security/ws-wpa-psk.ui
@@ -12,7 +12,7 @@
         <property name="can_focus">False</property>
         <property name="n_rows">3</property>
         <property name="n_columns">2</property>
-        <property name="column_spacing">12</property>
+        <property name="column_spacing">6</property>
         <property name="row_spacing">6</property>
         <child>
           <object class="GtkLabel" id="wpa_psk_label">
diff --git a/panels/network/wireless-security/eap-method-fast.c b/panels/network/wireless-security/eap-method-fast.c
index ec1cd6455..98d5411a7 100644
--- a/panels/network/wireless-security/eap-method-fast.c
+++ b/panels/network/wireless-security/eap-method-fast.c
@@ -349,7 +349,7 @@ eap_method_fast_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          destroy,
-	                          "/org/freedesktop/network-manager-applet/eap-method-fast.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-fast.ui",
 	                          "eap_fast_notebook",
 	                          "eap_fast_anon_identity_entry",
 	                          FALSE);
diff --git a/panels/network/wireless-security/eap-method-leap.c b/panels/network/wireless-security/eap-method-leap.c
index 0087bdf24..57d8759b9 100644
--- a/panels/network/wireless-security/eap-method-leap.c
+++ b/panels/network/wireless-security/eap-method-leap.c
@@ -199,7 +199,7 @@ eap_method_leap_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          destroy,
-	                          "/org/freedesktop/network-manager-applet/eap-method-leap.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-leap.ui",
 	                          "eap_leap_notebook",
 	                          "eap_leap_username_entry",
 	                          FALSE);
diff --git a/panels/network/wireless-security/eap-method-peap.c b/panels/network/wireless-security/eap-method-peap.c
index 402887c87..f75631e10 100644
--- a/panels/network/wireless-security/eap-method-peap.c
+++ b/panels/network/wireless-security/eap-method-peap.c
@@ -349,7 +349,7 @@ eap_method_peap_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          destroy,
-	                          "/org/freedesktop/network-manager-applet/eap-method-peap.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-peap.ui",
 	                          "eap_peap_notebook",
 	                          "eap_peap_anon_identity_entry",
 	                          FALSE);
diff --git a/panels/network/wireless-security/eap-method-simple.c b/panels/network/wireless-security/eap-method-simple.c
index 80de96482..2287b99cd 100644
--- a/panels/network/wireless-security/eap-method-simple.c
+++ b/panels/network/wireless-security/eap-method-simple.c
@@ -296,7 +296,7 @@ eap_method_simple_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          destroy,
-	                          "/org/freedesktop/network-manager-applet/eap-method-simple.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-simple.ui",
 	                          "eap_simple_notebook",
 	                          "eap_simple_username_entry",
 	                          flags & EAP_METHOD_SIMPLE_FLAG_PHASE2);
diff --git a/panels/network/wireless-security/eap-method-tls.c b/panels/network/wireless-security/eap-method-tls.c
index ce6763b37..aa0491c56 100644
--- a/panels/network/wireless-security/eap-method-tls.c
+++ b/panels/network/wireless-security/eap-method-tls.c
@@ -449,7 +449,7 @@ eap_method_tls_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          NULL,
-	                          "/org/freedesktop/network-manager-applet/eap-method-tls.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-tls.ui",
 	                          "eap_tls_notebook",
 	                          "eap_tls_identity_entry",
 	                          phase2);
diff --git a/panels/network/wireless-security/eap-method-ttls.c b/panels/network/wireless-security/eap-method-ttls.c
index 2d9de48d6..efdc00676 100644
--- a/panels/network/wireless-security/eap-method-ttls.c
+++ b/panels/network/wireless-security/eap-method-ttls.c
@@ -398,7 +398,7 @@ eap_method_ttls_new (WirelessSecurity *ws_parent,
 	                          fill_connection,
 	                          update_secrets,
 	                          destroy,
-	                          "/org/freedesktop/network-manager-applet/eap-method-ttls.ui",
+	                          "/org/gnome/ControlCenter/network/eap-method-ttls.ui",
 	                          "eap_ttls_notebook",
 	                          "eap_ttls_anon_identity_entry",
 	                          FALSE);
diff --git a/panels/network/wireless-security/ws-dynamic-wep.c b/panels/network/wireless-security/ws-dynamic-wep.c
index 810a10565..c267cac4d 100644
--- a/panels/network/wireless-security/ws-dynamic-wep.c
+++ b/panels/network/wireless-security/ws-dynamic-wep.c
@@ -109,7 +109,7 @@ ws_dynamic_wep_new (NMConnection *connection,
 	                                 fill_connection,
 	                                 update_secrets,
 	                                 destroy,
-	                                 "/org/freedesktop/network-manager-applet/ws-dynamic-wep.ui",
+	                                 "/org/gnome/ControlCenter/network/ws-dynamic-wep.ui",
 	                                 "dynamic_wep_notebook",
 	                                 NULL);
 	if (!parent)
diff --git a/panels/network/wireless-security/ws-leap.c b/panels/network/wireless-security/ws-leap.c
index 0d2ce75bf..35b2e034b 100644
--- a/panels/network/wireless-security/ws-leap.c
+++ b/panels/network/wireless-security/ws-leap.c
@@ -154,7 +154,7 @@ ws_leap_new (NMConnection *connection, gboolean secrets_only)
 	                                 fill_connection,
 	                                 update_secrets,
 	                                 NULL,
-	                                 "/org/freedesktop/network-manager-applet/ws-leap.ui",
+	                                 "/org/gnome/ControlCenter/network/ws-leap.ui",
 	                                 "leap_notebook",
 	                                 "leap_username_entry");
 	if (!parent)
diff --git a/panels/network/wireless-security/ws-wep-key.c b/panels/network/wireless-security/ws-wep-key.c
index c7472ca39..8b78515b1 100644
--- a/panels/network/wireless-security/ws-wep-key.c
+++ b/panels/network/wireless-security/ws-wep-key.c
@@ -264,7 +264,7 @@ ws_wep_key_new (NMConnection *connection,
 	                                 fill_connection,
 	                                 update_secrets,
 	                                 destroy,
-	                                 "/org/freedesktop/network-manager-applet/ws-wep-key.ui",
+	                                 "/org/gnome/ControlCenter/network/ws-wep-key.ui",
 	                                 "wep_key_notebook",
 	                                 "wep_key_entry");
 	if (!parent)
diff --git a/panels/network/wireless-security/ws-wpa-eap.c b/panels/network/wireless-security/ws-wpa-eap.c
index 2479a3ee2..f9a9be37e 100644
--- a/panels/network/wireless-security/ws-wpa-eap.c
+++ b/panels/network/wireless-security/ws-wpa-eap.c
@@ -110,7 +110,7 @@ ws_wpa_eap_new (NMConnection *connection,
 	                                 fill_connection,
 	                                 update_secrets,
 	                                 destroy,
-	                                 "/org/freedesktop/network-manager-applet/ws-wpa-eap.ui",
+	                                 "/org/gnome/ControlCenter/network/ws-wpa-eap.ui",
 	                                 "wpa_eap_notebook",
 	                                 NULL);
 	if (!parent)
diff --git a/panels/network/wireless-security/ws-wpa-psk.c b/panels/network/wireless-security/ws-wpa-psk.c
index befe68b04..29064f4f3 100644
--- a/panels/network/wireless-security/ws-wpa-psk.c
+++ b/panels/network/wireless-security/ws-wpa-psk.c
@@ -182,7 +182,7 @@ ws_wpa_psk_new (NMConnection *connection, gboolean secrets_only)
 	                                 fill_connection,
 	                                 update_secrets,
 	                                 NULL,
-	                                 "/org/freedesktop/network-manager-applet/ws-wpa-psk.ui",
+	                                 "/org/gnome/ControlCenter/network/ws-wpa-psk.ui",
 	                                 "wpa_psk_notebook",
 	                                 "wpa_psk_entry");
 	if (!parent)
